locales/en.json

```diff
@@ -210,5 +210,15 @@
       "in-progress": "In progress",
       "not-started": "Not reviewed"
     }
+  },
+  "dashboards": {
+    "title": "Dashboard",
+    "fra-kaare": {
+      "title": "Fra Kåre",
+      "largeChurches": "Large Churches",
+      "smallChurches": "Small Churches",
+      "oneEpisodeDescription": "Listened to at least one episode",
+      "allEpisodesDescription": "Listened to every episode"
```

Comment:
the description for one episode and all episodes will probably change relatively often. Therefore I'd prefer to not translate it.